modules/http-helper/dummy_server.go

```diff
@@ -46,6 +46,39 @@ func RunDummyServerE(t *testing.T, text string) (net.Listener, int, error) {
 	return listener, port, err
 }
 
+// RunDummyHandlerServer runs a dummy HTTP server on a unique port that will serve given handlers. Returns the Listener for the server,
+// the port it's listening on, or an error if something went wrong while trying to start the listener. Make sure to call
+// the Close() method on the Listener when you're done!
+func RunDummyHandlerServer(t *testing.T, handlers map[string]func(http.ResponseWriter, *http.Request)) (net.Listener, int) {
```

Comment:
```suggestion
func RunDummyServerWithHandlers(t *testing.T, handlers map[string]func(http.ResponseWriter, *http.Request)) (net.Listener, int) {
```
I propose renaming the functions to `RunDummyServerWithHandlers` and `RunDummyServerWithHandlersE` as I feel it follows the other Terratest functions more closely and the verb-style function names in the go `http` package: https://golang.org/pkg/net/http/#example_ListenAndServe. What do you think?

Reply:
Makes sense :+1: Adapted to it.